user_guide_src/source/changelogs/v4.5.0.rst

```diff
@@ -72,6 +72,9 @@ Forge
 Others
 ------
 
+- Update entity to use interface for better extensibility.
+
```

Comment:
```suggestion
- **Entity:**
    - Added ``CodeIgniter\Entity\EntityInterface`` for better extensibility.
    - Now ``Entity`` implements ``EntityInterface``.
    - Now ``Model`` and ``Result`` classes use ``EntityInterface`` instead of ``Entity``.
```